user-interface/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectContentComponent.java

```diff
@@ -29,13 +35,39 @@ public class ProjectContentComponent extends Div {
   private static final long serialVersionUID = -1061134126086910532L;
   private final ProjectDetailsComponent projectDetailsComponent;
 
+  private final ProjectPurchaseService projectPurchaseService;
+  private Context context;
+
   public ProjectContentComponent(
-      @Autowired ProjectDetailsComponent projectDetailsComponent) {
+      @Autowired ProjectDetailsComponent projectDetailsComponent,
+      @Autowired ProjectPurchaseService projectPurchaseService) {
     Objects.requireNonNull(projectDetailsComponent);
+    this.projectPurchaseService = Objects.requireNonNull(projectPurchaseService);
+    UploadPurchaseDialog uploadPurchaseDialog = new UploadPurchaseDialog();
+    uploadPurchaseDialog.setOpened(false);
+    add(uploadPurchaseDialog);
+    Button uploadOffer = new Button("Upload offer");
+    uploadOffer.addClickListener(listener -> uploadPurchaseDialog.open());
+    uploadPurchaseDialog.subscribe(
```

Comment:
please rename the method to `addConfirmListener` to be in sync with the other dialogs.